"""fix""" en ninemanga

xtj-9182 opened the pull request:
not sure if this is a good idea but it works ¯\\\_(ツ)_/¯

I can do the same with DE and RU with wiemanga,com and mangarussia.com respectively, since ninemanga links to these sites. I don't have a fix for IT ninemanga though ...

raulhaag commented:
before to add, does it works for update current collection?

xtj-9182 commented:
ok let me test that real quick

xtj-9182 commented:
unfortunately it's not that simple

the chapter paths are sadly different;
Ninemanga en
taadd
``` html
<a class="chapter_list_a" href="/chapter/Fairy%20Tail/865127.html" title="Fairy Tail 531">Fairy Tail 531</a>
<a href="/chapter/FairyTail531/865127/">Fairy Tail 531</a>
```

xtj-9182 commented:
Alternatively I can also revert the en ninemanga change and add taadd as a separate source in addition to wiemanga.com and mangarussia.com.

raulhaag commented:
````sql 
update capitulos set path = replace(path,"ninemanga.com/","taadd.com/") where id in(select capitulos.id from manga join capitulos on capitulos.manga_id = manga.id where manga.server_id = 23)
```
that do the trick for en change the 23 for the serverid ;-) (add a db update trigger changing the db version), add www if you left that on server but no needed

raulhaag commented:
yes, better add new servers and make announce that next version those go to be deprecated

xtj-9182 commented:
ok I'm done here.

I wanted to add wiemanga and mangarussia but I can't get images from them and they seem to give me ads too I have no idea why taadd works.

Anyway so to sum it up
I basically deprecated en ninemanga and added taadd as replacement, I wanted to add wiemanga and mangarussia as de and ru ninemanga replacements but I can't get them to work so idk what to do about that. de and ru ninemanga still give me ads :(. for some reason it ninemanga works for me sometimes.

I also fixed MyMangaIO and ReadMangaToday.

raulhaag commented:
merged
